src/KafkaFlow/Middlewares/Serializer/SerializerProducerMiddleware.cs

```diff
@@ -38,18 +38,21 @@ public async Task Invoke(IMessageContext context, MiddlewareDelegate next)
     {
         await _typeResolver.OnProduceAsync(context);
 
-        byte[] messageValue;
+        byte[] messageValue = null;
```

Comment:
Is null the safest here? What about 

`byte[] messageValue = Array.Empty<byte>();`?

Reply:
Hello esskar!

I may be understanding this wrong, but I believe we want to produce record entries with null message values (the so called tombstone records), so in that case it's better consider it as null? What do you think?

Reply:
Hello again esskar! 

In spite of the fact that null initialization was ok in our tests, we decided that your suggestion was actually more bullet proof. I've changed the code to apply that and so I'll close this now. 

Thank you! 🙇